src/backend/base/langflow/components/brightdata/brightdata_search_engine.py

```
@@ -0,0 +1,140 @@
+            if not query:
+                error_msg = "Search query is required"
+                return Data(
+                    text=error_msg,
+                    data={
+                        "status": "error",
+                        "error": error_msg
+                    }
+                )
+            
+            search_url = self._build_search_url(self.engine, query)
+            
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)